sdk/servicebus/Azure.Messaging.ServiceBus/tests/Infrastructure/ServiceBusScope.cs

```diff
@@ -0,0 +1,627 @@
+                                                            double baseJitterSeconds = RetryBaseJitterSeconds) =>
+           Policy<T>
+               .Handle<Exception>(ex => ShouldRetry(ex))
+               .WaitAndRetryAsync(maxRetryAttempts, attempt => CalculateRetryDelay(attempt, exponentialBackoffSeconds, baseJitterSeconds));
+
+        /// <summary>
+        ///   Creates the retry policy to apply to a management operation.
+        /// </summary>
+        ///
+        /// <param name="maxRetryAttempts">The maximum retry attempts to allow.</param>
+        /// <param name="exponentialBackoffSeconds">The number of seconds to use as the basis for backing off on retry attempts.</param>
+        /// <param name="baseJitterSeconds">TThe number of seconds to use as the basis for applying jitter to retry back-off calculations.</param>
+        ///
+        /// <returns>The retry policy in which to execute the management operation.</returns>
+        ///
+        private static IAsyncPolicy CreateRetryPolicy(int maxRetryAttempts = RetryMaximumAttempts,
+                                                      double exponentialBackoffSeconds = RetryExponentialBackoffSeconds,
+                                                      double baseJitterSeconds = RetryBaseJitterSeconds) =>
+            Policy
+                .Handle<Exception>(ex => ShouldRetry(ex))
+                .WaitAndRetryAsync(maxRetryAttempts, attempt => CalculateRetryDelay(attempt, exponentialBackoffSeconds, baseJitterSeconds));
+
+        /// <summary>
+        ///   Determines whether the specified HTTP status code is considered eligible to retry
+        ///   the associated operation.
+        /// </summary>
+        ///
+        /// <param name="statusCode">The status code to consider.</param>
+        ///
+        /// <returns><c>true</c> if the status code is eligible for retries; otherwise, <c>false</c>.</returns>
+        ///
+        private static bool IsRetriableStatus(HttpStatusCode statusCode) =>
+            ((statusCode == HttpStatusCode.Unauthorized)
+                || (statusCode == ((HttpStatusCode)408))
+                || (statusCode == HttpStatusCode.Conflict)
+                || (statusCode == ((HttpStatusCode)429))
+                || (statusCode == HttpStatusCode.InternalServerError)
+                || (statusCode == HttpStatusCode.ServiceUnavailable)
+                || (statusCode == HttpStatusCode.GatewayTimeout));
+
+        /// <summary>
+        ///   Determines whether the specified exception is considered eligible to retry the associated
+        ///   operation.
+        /// </summary>
+        ///
+        /// <param name="ex">The exception to consider.</param>
+        ///
+        /// <returns><c>true</c> if the exception is eligible for retries; otherwise, <c>false</c>.</returns>
+        ///
+        private static bool ShouldRetry(Exception ex) =>
+            ((IsRetriableException(ex)) || (IsRetriableException(ex?.InnerException)));
```

Comment:
nit: some extra parens here

Reply:
That formatting is intentional.   I prefer to group explicitly.